Docs: Move image renderer doc to its own section and add configuration doc (#38759)
grafana/grafana: docs/sources/image-rendering/troubleshooting.md

+++
title = "Troubleshooting"
description = "Image rendering troubleshooting"
keywords = ["grafana", "image", "rendering", "plugin", "troubleshooting"]
weight = 100
+++

# Troubleshoot image rendering

> In order to troubleshoot the image renderer, you can access different kind of logs:

Reviewer: ```suggestion
To troubleshoot the image renderer, use the information available in different kind of logs.
```

- Enable debug log messages for rendering in the Grafana configuration file and inspect the Grafana server log.

```bash
[log]
filters = rendering:debug

> ```

Reviewer: Move down to own section "Enable debug mode".

Author: Tried something, let me know what you think about this new section.
